src/kem/kyber/constant_time_ops.rs

```diff
@@ -5,8 +5,12 @@ use super::constants::SHARED_SECRET_SIZE;
 
 #[inline]
 fn is_non_zero(value: u8) -> u8 {
```

Comment:
This is a function that you could test well with quickcheck or proptest. How about adding such a test for it?
Also, it's a good target for a small, easy F* proof.
But you could do that in a follow up.

Reply:
Added some hax attributes as discussed in the call.